client/my-sites/domains/domain-search/index.jsx

```diff
@@ -158,7 +162,23 @@ class DomainSearch extends Component {
 				<span>
 					<div className="domain-search__content">
 						<PlansNavigation cart={ this.props.cart } path={ this.props.context.path } />
-
+						{ hasCustomDomain( selectedSite ) && (
+							<Notice
+								status="is-warning"
+								showDismiss={ false }
+								text={
+									translate( "You're purchasing an additional domain for %(title)s.", {
+										args: { title: selectedSiteTitle },
+									} ) +
+									' ' +
```

Comment:
I now see this. No need to split them. Please make it a single string. :)

Reply:
Would this not run the risk of translators missing the space at the end of each sentences? I recall that happening before and since then, I've always split them. :)

Reply:
Not really. This is a single message and should be translated as such. We have plenty of these, even in this file: https://github.com/Automattic/wp-calypso/blob/b349ba2c29137398527ea5e1864bd64cf408681b/client/my-sites/domains/domain-search/index.jsx#L151

Reply:
Ah, I understand now. Updated, thanks! :)